src/componentes/unmounted/lista-nominal/print/aux/StatusAtendimentoOdontologico.jsx

```
@@ -0,0 +1,31 @@
+  1: estilos.tagVerdeComIcone,
+  2: estilos.tagVermelhaComIcone,
+  3: estilos.tagNulaComIcone,
+}
+
+export const StatusAtendimentoOdontologico = ({
+  value,
+  identificacao_atendimento_odontologico,
+  orientacao = "paisagem"
+})=> {
+  const descricao = useMemo(() => {
+    return identificacao_atendimento_odontologico.find(item => item.id_atendimento_odontologico == value).atendimento_odontologico_descricao;
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)